packages/libs/components/src/stories/plots/Barplot.stories.tsx

```diff
@@ -185,3 +188,47 @@ LogScale.args = {
     width: '750px',
   },
 };
+
+// demo plot resize with draggable panel
+export const PlotResizeWithDraggablePanel: Story<BarplotProps> = (args) => {
+  const panelTitle = 'Panel 1';
+  const draggablePanelHeight = 600;
+  const draggablePanelWidth = 900;
+
+  const [panelDimension, setPanelDimension] = useState<HeightAndWidthInPixels>({
+    height: draggablePanelHeight,
+    width: draggablePanelWidth,
+  });
+
+  return (
+    <DraggablePanel
+      defaultPosition={{ x: 200, y: 200 }}
+      confineToParentContainer
+      key={panelTitle}
+      isOpen
+      panelTitle={panelTitle}
+      showPanelTitle={true}
+      styleOverrides={{
+        resize: 'both',
+      }}
+      onPanelResize={(dimensions: HeightAndWidthInPixels) => {
```

Comment:
I think we need to use `useCallback` on the function here and put the dependency back (see below)

Reply:
`useState` setters are guaranteed to be stable